planet_explorer/gui/pe_show_curl_dialog.py

```diff
@@ -4,7 +4,7 @@
 from qgis.PyQt import uic
 from qgis.PyQt.QtGui import QGuiApplication
 
-from ..pe_analytics import analytics_track
+from ..pe_analytics import analytics_track, CURL_REQUEST_COPIED
```

Comment:
In L15 down below can we put the `PLANET_API_KEY` in quotes - this will allow users to run the script without modification.